src/Redistribution.sol

```diff
@@ -132,9 +128,9 @@ contract Redistribution is AccessControl, Pausable {
     }
 
     /**
-     * @dev Emitted when the winner of a round is selected in the claim phase.
+     * @dev Emitted when the winner of a round is selected in the claim phase with current round number
      */
-    event WinnerSelected(Reveal winner);
+    event WinnerSelected(Reveal winner, uint256 roundNumber);
```

Comment:
on second thoughts i think we should remove the round numbers - they are trivial to calculate from blockheight and leaving the events untouched will reduce the burden on users of the contract events api